trunk/NDHMS/Routing/module_RT.F

```diff
@@ -796,6 +798,36 @@ subroutine LandRT_ini(did)
 
        end select
 
+     else if (rt_domain(did)%q_type(lake_index) == 2) then
+
```

Comment:
regarding the [singleton timeslice reader](https://github.com/NCAR/wrf_hydro_nwm_public/pull/259/files?file-filters%5B%5D=.F&file-filters%5B%5D=.inc&file-filters%5B%5D=.json&file-filters%5B%5D=.namelist&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#r297871100) 

seems like this would be the place to create a timeslice reader and pass it to the hybrid level pool. I dont know exactly how that would work, but it would be way more explicit than they way it currently works.

Just a musing.

Reply:
We did this to hide implementation details outside of the reservoir module given that the singleton timeslice reader is only used by reservoirs.